e2e/pageobjects/ide/TopMenu.ts

```diff
@@ -27,8 +29,15 @@ export class TopMenu {
         }
     }
 
+    public async selectOption(topMenuButtonText: string, submenuItemtext: string) {
+        await this.clickOnTopMenuButton(topMenuButtonText);
+        await this.clickOnSubmenuItem(submenuItemtext);
+    }
+
     public async clickOnTopMenuButton(buttonText: string, timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
         const buttonLocator: By = this.getTopMenuButtonLocator(buttonText);
+
+        await this.ide.closeAllNotifications();
```

Comment:
I guess this is done because element with locator `buttonLocator` is not clickable at the time... WDYT about calling `closeAllNotifications`  only when this happens? I think it would make sense to leave this as is if the notifications will be there in 100% time. Is this the case?